lib/pages/timetable/view/events/add_event_view.dart

```diff
@@ -686,3 +706,18 @@ extension LocalTimeConversion on LocalTime {
 extension TimeOfDayConversion on TimeOfDay {
   LocalTime toLocalTime() => LocalTime(hour, minute, 0);
 }
+
+extension LocalDateComparisons on LocalDate {
+  bool isDuring(AllDayUniEvent semester) {
+    if (DateInterval(semester.startDate, semester.endDate).contains(this)) {
+      return true;
+    }
+    return false;
```

Comment:
Uhm, sure, but why not just:
```suggestion
    return DateInterval(semester.startDate, semester.endDate).contains(this));
```